examples/external_decorators/README.md

```diff
@@ -0,0 +1,21 @@
+# External decorators demo
+
+This example is in response to this [GitHub issue](https://github.com/DAGWorks-Inc/hamilton/issues/50). However, it showcases more general mechanisms of how to parse "Hamilton  modules" and dynamically alter source code.
```

Comment:
Mention the overall goal/motivation first, then that it is in response to the issue. Good to have a "why".

Also worth calling it something other than "External decorators" -- E.G. "Reusing Hamilton Functions without depending on Hamilton" or something like that.